experiments/calibration/shared/observation_data_global.jl

```diff
@@ -0,0 +1,26 @@
+import ClimaAnalysis
```

Comment:
One thing you need to watch out for is where the data is defined in time.

For example, 2024-02-01 could mean average for the month of february or for the month of january (as I think it's our case right now).

What we do in the leaderboard code is moving everything to the first of the month to line things up
(https://github.com/CliMA/ClimaCoupler.jl/blob/main/experiments/ClimaEarth/leaderboard/data_sources.jl)

However, this depends on your data sources, so make sure you understand them well.